doc/dev/deduplication.rst

```diff
@@ -157,25 +157,46 @@ How to use deduplication
 Ceph provides deduplication using RADOS machinery.
 Below we explain how to perform deduplication. 
 
+Prerequisite
+------------
+
+If the Ceph cluster is started from Ceph mainline, users need to check
+``ceph-test`` package which is including ceph-dedup-tool is installed.
+
+Deatiled Instructions
+---------------------
+
+Users can use ceph-dedup-tool with ``estimate``, ``sample-dedup``, 
+``chunk-scrub``, and ``chunk-repair`` operations. To provide users with
+high flexibility, we have enabled necessary operations through
+ceph-dedup-tool, and we recommend using the following operations freely
+by using any types of scripts.
```

Comment:
@zdover23: The sentence beginning "To provide users with high flexibility" requires rewording.

Reply:
This sentence has been changed to "To provide better convenience for users,".